client/src/context/useBookingContext.tsx

```diff
@@ -0,0 +1,25 @@
+import { useContext, createContext, FunctionComponent, useState, useEffect } from 'react';
+import { Booking } from '../interface/Booking';
+import sortedBookings from '../helpers/utils/sortBookings';
+
+interface IBookingContext {
+  bookings: Booking[];
+}
+
+export const BookingContext = createContext<IBookingContext>({
+  bookings: sortedBookings,
+});
+
+export const BookingProvider: FunctionComponent = ({ children }): JSX.Element => {
+  const [bookings, setBookings] = useState<Booking[]>([]);
+
+  useEffect(() => {
+    setBookings(sortedBookings);
+  }, [bookings]);
```

Comment:
This dependency array doesn't seem right 🤔 Did you mean `setBookings`?

Reply:
@ugur-akin, What if I can keep the dependency array empty?

Reply:
@gagandeep95 Since the  `setBookings` function is not subject to change, you can leave your dependency array empty in this case